0001/README.md

```diff
@@ -0,0 +1,398 @@
+---
+id: 0001
+title: Request for discussion HOWTO
+status: discussion
+pr: 2
```

Comment:
`pr: 2` This can not be known when creating PR because of race conditions. I don't see value in the numeric index value here.  On the other hand, permalinks to the PR and the Github file URL (in the default branch) would be useful.  These pieces of metadata could be added in commits or PRs that transition the `status` to later stages.  I don't think we need to try to capture them in the initial PR.